transport/spi/pom.xml

```diff
@@ -45,11 +45,13 @@
         <dependency>
             <groupId>org.jmock</groupId>
             <artifactId>jmock</artifactId>
+            <scope>provided</scope>
```

Comment:
It doesn't need to be done now, but we should consider moving these mocks into a test-jar and using that test jar dependency in other builds that need the "matchers".